src/sequences/SequenceEntity.js

```diff
@@ -14,6 +14,23 @@ export const sequencedActivityClasses = {
 };
 
 export class SequenceEntity extends Entity {
+	/**
+	 * @return An object with indices: completed, total, optionalViewed, optionalTotal, isCompleted
+	 */
+	completion() {
+		const subEntities = this._entity && this._entity.getSubEntitiesByRel && this._entity.getSubEntitiesByRel(Rels.completion);
+		const subEntity = subEntities.pop();
+
+		if (!subEntity) {
+			return;
+		}
+
+		const completionProperties = subEntity.properties || {};
+		completionProperties.isCompleted = !!(subEntity.hasClass && subEntity.hasClass(Classes.activities.complete));
```

Comment:
what `subEntity.hasClass` returns, function ?

Reply:
functions are `truthy` so it would then run the method.

Reply:
nevermind for my comment above^, line 28 explains it.